effects/cats-ce2/src/main/scalajs/sttp/client3/impl/cats/FetchCatsBackend.scala

```diff
@@ -48,5 +48,5 @@ object FetchCatsBackend {
     *
     * See [[SttpBackendStub]] for details on how to configure stub responses.
     */
-  def stub[F[_]: Concurrent: ContextShift]: SttpBackendStub[F, Any] = SttpBackendStub(new CatsMonadAsyncError)
```

Comment:
Thanks for the PR - following suit for other stub backends, I think it would be sufficient to simply type this using the `WebSockets` capability. Then you can upcast to `Any` because of variance. See e.g.: https://github.com/softwaremill/sttp/blob/d0fb7e962ae3efc0b9e808bb273bee199de8f66d/effects/fs2/src/main/scalajvm/sttp/client3/httpclient/fs2/HttpClientFs2Backend.scala#L173

Reply:
No no, thank you for the awesome lib, I'm just trying to give back a little 😄 

For the reply, I'm not sure I get it. The docs for the second type parameter in `SttpBackend` say

> P – Capabilities supported by this backend, in addition to sttp.capabilities.Effect. This might be Any (no special capabilities), sttp.capabilities.Streams (the ability to send and receive streaming bodies) or sttp.capabilities.WebSockets (the ability to handle websocket requests).

So should the change be, this? (then we only support `Websockets`? Not sure if `FetchCatsBackend` can support anything else?)
```scala
  def stub[F[_]: Async, WebSockets]: SttpBackendStub[F, WebSockets] = SttpBackendStub(implicitly)
```
or there should ne now change at all 😄

Reply:
Yes, it should be changed as you described. In general this should be the maximal set of capabilities - you can always upcast a backend to narrow the supported capabilities list. In case of cats, it's only WebSockets, no streaming is supported.

Reply:
Oh okay understood, thank you.

Code changed 😄